docs/integrations/amazon-aws/amazon-chime.md

```diff
@@ -0,0 +1,29 @@
+---
+id: amazon-chime
+title: Amazon Chime
+description: Learn about the collection process for the Amazon Chime service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/amazon-chime-logo.png')} alt="Thumbnail icon" width="50"/>
+
+Amazon Chime is a communications service that transforms online meetings with an application that is secure and comprehensive. Amazon Chime works across your devices so that you can stay connected. You can use Amazon Chime for online meetings, video conferencing, calls, and chat. You can also share content inside and outside of your organization. Amazon Chime is a fully managed service that runs securely on the AWS cloud, which frees IT from deploying and managing complex infrastructures. For more details, refer to the [AWS documentation](https://docs.aws.amazon.com/chime/latest/ag/what-is-chime.html).
+
+## Log and Metric types
+* [CloudWatch Metrics](https://docs.aws.amazon.com/chime/latest/ag/monitoring-cloudwatch.html)
+* [CloudWatch Logs](https://docs.aws.amazon.com/chime/latest/ag/monitoring-cloudwatch.html#cw-logs)
+* [CloudTrail Logs](https://docs.aws.amazon.com/chime/latest/ag/cloudtrail.html)
+
+
+## Setup
+You can collect the logs and metrics for Sumo Logic's Amazon Chime integration by following the below steps.
+
+### Configure metrics collection
+* Collect **CloudWatch Metrics** with namespace `AWS/ChimeVoiceConnector` using the [AWS Kinesis Firehose for Metrics](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source/) source. For `AWS/ChimeVoiceConnector` metrics and dimensions, refer to [Amazon Chime CloudWatch metrics](https://docs.aws.amazon.com/chime/latest/ag/monitoring-cloudwatch.html).
+### Configure logs collection
+* Collect [Amazon CloudWatch Logs](https://docs.aws.amazon.com/chime/latest/ag/monitoring-cloudwatch.html#cw-logs) using [AWS Kinesis Firehose for Logs](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/) source. Media quality metric logs: You can opt to receive media quality metric logs for your Amazon Chime Voice Connector. When you do, Amazon Chime sends detailed, per-minute metrics for all of your Amazon Chime Voice Connector calls to a CloudWatch Logs log group that is created for you. The log group name is /aws/ChimeVoiceConnectorLogs/${VoiceConnectorID}. 
```

Comment:
```suggestion
* Collect [Amazon CloudWatch Logs](https://docs.aws.amazon.com/chime/latest/ag/monitoring-cloudwatch.html#cw-logs) using [AWS Kinesis Firehose for Logs](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/) source. You can opt to receive media quality metric logs for your Amazon Chime Voice Connector. When you do, Amazon Chime sends detailed, per-minute metrics for all of your Amazon Chime Voice Connector calls to a CloudWatch Logs log group that is created for you. The log group name is `/aws/ChimeVoiceConnectorLogs/${VoiceConnectorID}`. 
```

Reply:
There's no need for "Media quality metric logs:" since you mention it in the following sentence.